finality-provider/service/fastsync.go

```diff
@@ -62,14 +62,6 @@ func (fp *FinalityProviderInstance) FastSync(startHeight, endHeight uint64) (*Fa
 				fp.metrics.IncrementFpTotalBlocksWithoutVotingPower(fp.GetBtcPkHex())
 				continue
 			}
-			// check whether the randomness has been committed
```

Comment:
question: we can stop checking the randomness has been committed, because we don't need to send randomness for past blocks?

Reply:
This is because in [ADR-024](https://github.com/babylonlabs-io/pm/blob/main/adr/adr-024-timestamping-public-randomness.md), a fp can only have voting power if it has BTC-timestamped pub rand for the height. At Line 57, we have already checked whether the fp has voting power. This indicates whether the fp has timestamped pub rand or not, so we don't need to check randomness here